scripts/system/more/app-more.js

```diff
@@ -0,0 +1,145 @@
+//    
+(function() {
+    var ROOT = Script.resolvePath('').split("app-more.js")[0];
+    var APP_NAME = "MORE...";
+    var APP_URL = ROOT + "more.html";
+    var APP_ICON_INACTIVE = ROOT + "appicon_i.png";
+    var APP_ICON_ACTIVE = ROOT + "appicon_a.png";
+    var Appstatus = false;
+    var lastProcessing = {
+            "action": "",
+            "script": ""
+        };
+    
+    var tablet = Tablet.getTablet("com.highfidelity.interface.tablet.system");
+    tablet.screenChanged.connect(onScreenChanged);
+    var button = tablet.addButton({
+        text: APP_NAME,
+        icon: APP_ICON_INACTIVE,
+        activeIcon: APP_ICON_ACTIVE
+    });
+    
+    
+    function clicked() {
+        if (Appstatus) {
+            tablet.webEventReceived.disconnect(onMoreAppWebEventReceived);
+            tablet.gotoHomeScreen();
+            Appstatus = false;
+        } else {
+            tablet.gotoWebScreen(APP_URL);            
+            tablet.webEventReceived.connect(onMoreAppWebEventReceived);        
+            Appstatus = true;
+        }
+            
+        button.editProperties({
+            isActive: Appstatus
+        });
+
+    }
+    
+    button.clicked.connect(clicked);
+
+    function sendRunningScriptList() {
+        var currentlyRunningScripts = ScriptDiscoveryService.getRunning();
+        var newMessage = "RSL4MOREAPP:";
+        var runningScriptJson;
+        for (var j = 0; j < currentlyRunningScripts.length; j++) {
+            runningScriptJson = currentlyRunningScripts[j].url;
+            if (runningScriptJson.indexOf("https://kasenvr.github.io/community-apps/applications") !== -1) {
+                newMessage = newMessage + "_" + runningScriptJson;
+            }
+        }
+
+        tablet.emitScriptEvent(newMessage);
+    }
+
+    function onMoreAppWebEventReceived(eventz) {
+        
+        if (typeof eventz === "string") {
+            var eventzget = JSON.parse(eventz);
+            
+            //print("EVENT ACTION: " + eventzget.action);
+            //print("EVENT SCRIPT: " + eventzget.script);
+            
+            if (eventzget.action === "installScript") {
+                
+                if (lastProcessing.action === eventzget.action && lastProcessing.script === eventzget.script) {
+                    return;
+                } else {
+                    ScriptDiscoveryService.loadOneScript(eventzget.script);
+                
+                    lastProcessing.action = eventzget.action;
+                    lastProcessing.script = eventzget.script;
+                    
+                    Script.setTimeout(function() {
+                        sendRunningScriptList(); 
+                    }, 1500);
+                }
+            }
+
+            if (eventzget.action === "uninstallScript") {
+                
+                if (lastProcessing.action === eventzget.action && lastProcessing.script === eventzget.script) {
```

Comment:
this if return else thing seems a bit unlogical, it could be reduced to a single if statement since nothing else happens